packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -172,7 +172,7 @@ export class VaRadio {
   render() {
     const { label, required, error } = this;
     return (
-      <Host role="radiogroup">
+      <Host role="radiogroup" aria-invalid={error ? 'true' : 'false'}>
```

Comment:
In addition to [`radiogroup` being an associated role for `aria-invalid`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid#associated_roles), it seems like [this pattern has been seen elsewhere](https://dsva.slack.com/archives/C023ZKDJJK1/p1657907719138989?thread_ts=1657892156.507549&cid=C023ZKDJJK1).